test/hotspot/jtreg/serviceability/jvmti/events/VMDeath/AllocatingInVMDeath/libTestAllocatingInVMDeath.cpp

```diff
@@ -37,6 +37,10 @@ cbVMDeath(jvmtiEnv* jvmti, JNIEnv* jni) {
     fatal(jni, "Can't find upCall method.");
   }
   jni->CallStaticObjectMethod(clz, mid);
+  if (jni->ExceptionOccurred()) {
+    jni->ExceptionDescribe();
+    fatal(jni, "Unexpected exception occured in cbVMDeath.");
```

Comment:
Nit: Typo: `s/occured/occurred/`
Also, this message is confusing. Unexpected exception occurred in Java upcall made from the cbVMDeath.
Maybe something like this: "cbVMDeath: unexpected exception occurred in Java upcall method".